### PR TITLE
Reduce NGCC SOEC Notebook Runtime

### PR DESCRIPTION
**Pull request by bpaul4**
## Proposed changes:
- Update solver settings to reduce runtime of NGCC-SOEC notebook, to resolve intermittent timeout failures on CI runs.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by andrewlee94 — APPROVED** [review]
One comment, but it is probably beyond the scope of this PR.

File: src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb [hunk 1]
```diff
@@ -368,143 +370,83 @@
...
+      "  43  5.0425664e+01 1.69e-02 7.72e+03  -1.0 5.20e+01  -3.8 9.60e-01 1.00e+00f  1\n",
+      "  44  5.0442942e+01 1.55e+00 3.21e+04  -1.0 1.19e+03  -4.3 5.68e-02 2.21e-01f  1\n",
+      "  45  5.0437736e+01 4.84e-01 8.61e+03  -1.0 1.33e+02  -4.8 1.00e+00 1.00e+00h  1\n",
+      "  46  5.0448684e+01 4.75e-01 5.14e+04  -1.0 2.24e+02  -5.2 6.34e-01 1.00e+00h  1\n",
+      "  47  5.0450147e+01 9.90e-02 2.98e+02  -1.0 2.46e+02  -5.7 1.00e+00 1.00e+00h  1\n",
+      "  48  5.0452046e+01 4.37e-03 7.28e+01  -1.7 1.51e+02  -6.2 1.00e+00 1.00e+00h  1\n",
+      "  49  5.0456539e+01 2.39e-02 8.51e+01  -1.7 3.02e+02  -6.7 1.00e+00 1.00e+00h  1\n",
       "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
-      " 100  5.0555724e+01 1.46e-03 7.31e+03  -3.8 9.30e+01  -7.4 7.27e-01 1.00e+00h  1\n",
+      "  50  5.0462280e+01 3.11e-02 2.31e+01  -1.7 7.32e+02  -7.2 1.00e+00 1.00e+00h  1\n",
```
[the remark this answers is not in the document]
Comment (eslickj) [reply src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb:429]: I'm not really sure what's happening, but somehow this isn't solving right.  It used to solve pretty cleanly.  Maybe it's some recent framework changes.  I'll take a closer look at this.

[the remark this answers is not in the document]
Comment (eslickj) [reply src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb:429]: Ok, at least for me the model wasn't solving at the cost optimization step.  It was just terminating on max iterations.  The cost estimation variables and constrains are really badly scaled.  I added scale factors to the notebook.  I should have it sorted out soon. I'm not sure at what point things when bad but I guess I should also add some asserts to make sure things don't go bad again.

[the remark this answers is not in the document]
Comment (eslickj) [reply src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb:429]: After adding the capital cost estimate models now there are two extra degrees of freedom aside from the scaling issues.  I guess this is due to costing framework changes?  May have to call for costing expert help.

[the remark this answers is not in the document]
Comment (bpaul4) [reply src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb:429]: Thank you @eslickj for taking a closer look at the model. Would it be convenient to share your version with me offline, or push it to this PR? I can take a look at the DOF and any required changes to the costing.

[the remark this answers is not in the document]
Comment (eslickj) [reply src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb:429]: I think I have it working.  I'm going to do a little clean up and check the results against the report.  I guess the easiest thing is to just open a new PR and we can close this one.  I should have something up tomorrow morning.

**Comment by eslickj** [comment]
I have an alternative PR (#158) that gets more at the core problems, so closing this.

**Pull request closed without merging**